### PR TITLE
Add boundary based validation

### PR DESCRIPTION
**Pull request by JahnabDutta**
## Proposed Changes
- Change request body to `/opearate_asset` api to contain id of boundary-AssetBed realated to the given AssetBed and current CameraState.
- Added `validate_action` in `OnvifAsset` to validate  and kind of `relativeMove`, if the movement is within the range.
- Changed `handle_action` in `OnvifAsset` to ensure additional parameters in the request body do not affect the `handle_action` function.
- Created dummy `validate_action` in other asset integration modules for future use

### Associated Issue
- [Boundary-Based Camera Visual under Live Camera Feed Enhancement #5871](https://github.com/coronasafe/care_fe/issues/5871)
- Does not solve the issue completely but adds necessary backend validation.

## Merge Checklist

- [x] Tests added/fixed
- [ ] Update docs in `/docs`
- [x] Linting Complete
- [ ] Any other necessary step

_*Only PR's with test cases included and passing lint and test pipelines will be reviewed*_

@coronasafe/care-backend-maintainers @coronasafe/care-backend-admins

### CODE REVIEW

File: care/utils/assetintegration/onvif.py [hunk 1]
```diff
@@ -55,3 +58,42 @@ def handle_action(self, action):
             return self.api_post(self.get_url("relativeMove"), request_body)
 
         raise ValidationError({"action": "invalid action type"})
+
+    def validate_action(self, action):
```
Comment (sainak) [inline care/utils/assetintegration/onvif.py:62]: instead of returning a boolean it would be better to raise a proper validation error

**Review by sainak — CHANGES_REQUESTED** [review]
Revert changes on non-relevant files

File: data/medibase.json [hunk 2]
```diff
@@ -1144305,4 +1144305,4 @@
   "cims_class_link": "/india/drug/search?q=drugs+for+erectile+dysfunction&mtype=brand&code=10d",
   "cims_class": "Drugs for Erectile Dysfunction",
   "atc_classification": "G04BE03 - sildenafil ; Belongs to the class of drugs used in erectile dysfunction."
-}]
\ No newline at end of file
+}]
```
Comment (JahnabDutta) [inline data/medibase.json:1144308]: I do not know how this change came to be, should I also revert it? @sainak

File: requirements/base.txt [hunk 3]
```diff
@@ -58,3 +58,4 @@ pydantic==1.*
 jwcrypto==1.5.0 # https://github.com/latchset/jwcrypto/releases
 pycryptodome==3.16.0
 pycryptodomex==3.16.0
+pydantic==1.10.10
```
Comment (JahnabDutta) [inline requirements/base.txt:61]: do I need to remove this dependency as well? @sainak

**Pull request closed without merging**